src/east/helper_functions.py

```diff
@@ -174,13 +174,16 @@ def get_ncs_and_project_dir(west_dir_path: str) -> Tuple[str, str]:
     try:
         ncs = list(
             filter(
-                lambda project: project["repo-path"] == "sdk-nrf", manifest["projects"]
+                lambda project: project.get("repo-path", "") == "sdk-nrf",
+                manifest["projects"],
             )
         )
     except KeyError:
         # This can happen in the case where there is no sdk-nrf repo in the west yaml
         # file, project is probably using ordinary Zephyr.
         return None, project_path
+    if len(ncs) == 0:
+        return None, project_path
```

Comment:
I had to add this to make the following tests pass:
```
test_not_ncs_sdk_west_workplace
test_not_ncs_sdk_west_workplace[multi]
```

not really sure if what I did was correct

Reply:
Yep, that was fine